README.md

```diff
@@ -198,11 +207,82 @@ A numeric value that can be parsed by [`(float)` cast, equivalent to `floatval`]
 ###### `bool` default
 `true`, `on`, `yes` or `1` will result in true. Other values, as long as there is something, will result default false. (If there is nothing, the variable will not be optional)
 
+#### Example of using variables
+##### SQL file
+```sql
+-- #{ example
+-- #    { insert
+-- # 	  :foo string
+-- # 	  :bar int
+INSERT INTO example(
+	foo_column
+	bar_column
+) VALUES (
+	:foo
+	:bar
+);
+-- #    }
+-- #    { select
+-- # 	  :foo string
+-- # 	  :bar int
+SELECT * FROM example
+WHERE foo_column = :foo
+LIMIT :bar;
+-- #    }
+-- #}
+```
+##### Code
+```php
+// Example of using variable in insert statements
+$this->database->excuteInsert("example.insert", ["foo" => "sample text", "bar" => 123]);
+
+// Example of using variable in select statements
+$this->database->executeSelect("example.select", ["foo" => "sample text", "bar" => 1]);
```

Comment:
This does nothing if you don't handle its result.

Reply:
Imma make it echo the bar column

Reply:
# Added